include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -1187,5 +1187,11 @@ def err_forany_unexpected_token : Error<
 def err_forany_comma_or_parenthesis_expected : Error<
   "expected , or )">;
 
+def err_expected_list_of_types_expr_for_generic_function : Error<
```

Comment:
I would suggest changing the error message to "expected a list of type arguments for a generic function".